vocabularies/Org.OData.Core.V1.json

```diff
@@ -512,9 +511,13 @@
         },
         "ContentDispositionType": {
             "$Kind": "ComplexType",
+            "Type": {
+                "$Nullable": true,
+                "@Core.Description": "The disposition type of the binary or stream value, see [RFC 6266, Disposition Type](https://datatracker.ietf.org/doc/html/rfc6266#section-4.2)"
+            },
             "Filename": {
                 "$Nullable": true,
-                "@Core.Description": "The proposed filename for downloading the binary or stream value"
+                "@Core.Description": "The proposed filename for downloading the binary or stream value, see [RFC 6266, Disposition Parameter: 'Filename'](https://datatracker.ietf.org/doc/html/rfc6266#section-4.3)"
```

Comment:
`Filename` makes sense only if `Type = "Attachment"`.

Reply:
From RFC6266:

>   Depending on the disposition type, this information might be used
>   right away (in the "save as..." interaction caused for the
>   "attachment" disposition type), **or later on (for instance, when the
>   user decides to save the contents of the current page being
>   displayed).**

The "save later" use case combines `Type: inline` with a `Filename`.